snooty/types.py

```diff
@@ -365,6 +365,7 @@ def finish(
 class ProjectConfig:
     root: Path
     name: str
+    title: str = field(default="untitled")
```

Comment:
In this case, I think we actually _don't_ want to provide a default value. I believe that `title` should be a required field in the same way that `name` is, because I can't imagine a scenario where a site doesn't need a title. Thoughts @i80and?